azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/redux/middleware/handler/CallingMiddlewareActionHandler.kt

```diff
@@ -555,14 +556,26 @@ internal class CallingMiddlewareActionHandlerImpl(
                 }
 
                 if (callInfoModel.callStateError == null) {
-                    when (callInfoModel.callingStatus) {
-                        CallingStatus.CONNECTED, CallingStatus.IN_LOBBY -> {
-                            store.dispatch(NavigationAction.CallLaunched())
+                    if (configuration.callConfig?.callType == CallType.ONE_TO_N_OUTGOING) {
```

Comment:
Let's re-write it to avoid duplicate code in else branch.

Reply:
For instance:

val action: NavigationAction? = when (callInfoModel.callingStatus) {
    CallingStatus.DISCONNECTED -> NavigationAction.Exit()
    CallingStatus.CONNECTED -> NavigationAction.CallLaunched() 
    CallingStatus.CONNECTING -> {
        if (configuration.callConfig?.callType == CallType.ONE_TO_N_OUTGOING) NavigationAction.CallLaunched() else null
        }
    CallingStatus.IN_LOBBY -> {
        if (configuration.callConfig?.callType != CallType.ONE_TO_N_OUTGOING) NavigationAction.CallLaunched() else null
        }
        else -> null
}
 action?.let { store.dispatch(it) }